string obj_id
string bin_num
---
sensor_msgs/PointCloud pc
